### PR TITLE
Modifying Registry to use github container registry instead of docker

### PR DESCRIPTION
**Pull request by srprasanna**
Modifying Registry to use github container registry instead of dockerhub registry.

https://github.com/Sunbird-RC/community/issues/807

### CODE REVIEW

File: tools/cli/src/templates/.env
Comment (Sreejit-K) [inline tools/cli/src/templates/.env:1]: These Symbolic links will be used in the registry-cli build process.  Can we just keep them as it is and what was the purpose of removing these links?

**Review by Sreejit-K — CHANGES_REQUESTED** [review]

File: tools/cli/src/templates/.env
Comment (srprasanna) [reply tools/cli/src/templates/.env:1]: It was a mistake let me fix this.

Comment (Sreejit-K) [reply tools/cli/src/templates/.env:1]: Sure no worries. There are 3 files config.json, docker-compose.yml and the .env files for your reference.

**Review by Sreejit-K — APPROVED** [review]

File: build.sh
Comment (holashchand) [inline build.sh:1]: plz revert this

File: configure-dependencies.sh
Comment (holashchand) [inline configure-dependencies.sh:1]: plz revert this

File: deps/keycloak-mobile-number-login-spi/build.sh
Comment (holashchand) [inline deps/keycloak-mobile-number-login-spi/build.sh:1]: plz revert this

File: deps/keycloak-mobile-number-login-spi/mvnw
Comment (holashchand) [inline deps/keycloak-mobile-number-login-spi/mvnw:1]: plz revert this

File: deps/keycloak-mobile-number-login-spi/src/main/resources/META-INF/services/org.keycloak.authentication.AuthenticatorFactory
Comment (holashchand) [inline deps/keycloak-mobile-number-login-spi/src/main/resources/META-INF/services/org.keycloak.authentication.AuthenticatorFactory:1]: plz revert this

File: dockerPushToRepo.sh
Comment (holashchand) [inline dockerPushToRepo.sh:1]: plz revert this

File: java/mvnw
Comment (holashchand) [inline java/mvnw:1]: plz revert this

File: java/registry/src/main/java/dev/sunbirdrc/registry/service/impl/RegistryServiceImpl.java
Comment (holashchand) [inline java/registry/src/main/java/dev/sunbirdrc/registry/service/impl/RegistryServiceImpl.java:1]: plz revert this

File: services/notification-service/notification-service
Comment (holashchand) [inline services/notification-service/notification-service:1]: plz revert this

File: tools/cli/bin/registry
Comment (holashchand) [inline tools/cli/bin/registry:1]: plz revert this

**Review by holashchand — CHANGES_REQUESTED** [review]

**Pull request merged**